src/SUMMARY.md

```diff
@@ -61,11 +61,8 @@
    - [Microcode](./config/microcode.md)
 - [XBPS Package Manager](./xbps/index.md)
    - [Updating](./xbps/updating.md)
-   - [Packages](./xbps/packages/index.md)
-      - [General](./xbps/packages/management.md)
-      - [Downgrading](./xbps/packages/downgrading.md)
-      - [Building](./xbps/packages/building.md)
-      - [Finding Files](./xbps/packages/files.md)
+   - [Downgrading](./xbps/downgrading.md)
```

Comment:
Should the downgrading title be the same as the one in the page?

Reply:
It should indeed. :-) But i think the title of the page should be changed to simply "Downgrading", so i've just done that, and added an introductory sentence.